server/db/scorepads.js

```diff
@@ -28,8 +28,8 @@ function initScorepad(scorepad, players) {
  *
  * @param {function} onResult callback that is executed with the result documents from the database
  */
-function get(onResult) {
-  scorepadModel.find({}, (scorepads) => {
+function get(game, onResult) {
+  scorepadModel.find({game}, (scorepads) => {
```

Comment:
Hier dann auch `query`, ohne die geschweiften Klammern:
```suggestion
  scorepadModel.find(query, (scorepads) => {
```